tests/test_compiler.py

```diff
@@ -182,10 +190,10 @@ def test_get_version_map_raises_on_non_solidity_sources(compiler, vyper_source_p
 def test_compiler_data_in_manifest(project):
     manifest = project.extract_manifest()
 
-    compiler_0816 = [c for c in manifest.compilers if str(c.version) == "0.8.16"][0]
-    compiler_0812 = [c for c in manifest.compilers if str(c.version) == "0.8.12"][0]
-    compiler_0612 = [c for c in manifest.compilers if str(c.version) == "0.6.12"][0]
-    compiler_0426 = [c for c in manifest.compilers if str(c.version) == "0.4.26"][0]
+    compiler_0816 = [c for c in manifest.compilers if str(c.version) == "0.8.16+commit.07a7930e"][0]
```

Comment:
Does `manifest.compilers` return an iterator/generator?

Reply:
It is an unordered list.  I want to grab the entries that match the given versions, so like filter the lists and grab the single expected item.